src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -208,5 +225,42 @@ private NetworkPeerConnectionParameters CreateNetworkPeerConnectionParameters()
             param2.AddressFrom = myExternal;
             return param2;
         }
+
+        /// <summary>
+        /// Check if the client is allowed to connect based on certain criteria.
+        /// </summary>
+        /// <returns>When criteria is met returns <c>true</c>, to allow connection.</returns>
+        private bool AllowClientConnection(TcpClient tcpClient)
+        {
+            this.logger.LogTrace("()");
+
+            if (this.networkPeerDisposer.ConnectedPeersCount >= MaxConnectionThreshold)
+            {
+                this.logger.LogTrace("(-)[MAX_CONNECTION_THRESHOLD_REACHED]:'{0}'", MaxConnectionThreshold);
+                return false;
+            }
+
+            if (!this.initialBlockDownloadState.IsInitialBlockDownload())
+            {
+                this.logger.LogTrace("(-)[IBD_COMPLETE_ALLOW_CONNECTION]");
```

Comment:
logging style violation for every logging line in this method

Reply:
Yes, the entry log is missing parameter, all exit logs are missing the return value and the one online 239 has something else than the return value and we should even fix the wrong log from the old code that was copied here as there should not be square brackets but apostrophes around the curly zero